schema/shapes/stroke.json

```
@@ -0,0 +1,32 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Stroke",
+    "description": "Solid stroke",
+    "allOf": [
+        {
+            "$ref": "#/$defs/shapes/shape-style"
+        },
+        {
+            "$ref": "#/$defs/shapes/base-stroke"
+        },
+        {
+            "type": "object",
+            "properties": {
+                "ty": {
+                    "title": "Shape Type",
+                    "type": "string",
+                    "const": "st"
+                },
+                "c": {
+                    "title": "Color",
```

should we include a description as in [lottie-docs/schema](https://lottiefiles.github.io/lottie-docs/schema/#/$defs/shapes)?

```suggestion
                    "title": "Color",
                    "description": "Stroke color",
```